<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSettingsDlg</class>
 <widget class="QDialog" name="QSettingsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Buttons</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Radicals &amp;horizontal count</string>
            </property>
            <property name="buddy">
             <cstring>buttonsCnt</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="buttonsCnt">
            <property name="minimum">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>&amp;Kana horizontal count</string>
            </property>
            <property name="buddy">
             <cstring>buttonsCntKana</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="buttonsCntKana">
            <property name="minimum">
             <number>5</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Fonts</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_1">
            <property name="text">
             <string>&amp;Buttons</string>
            </property>
            <property name="buddy">
             <cstring>btnFontButtons</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="btnFontButtons">
            <property name="text">
             <string>Font...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="testBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>日本語能力試験</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&amp;Labels</string>
            </property>
            <property name="buddy">
             <cstring>btnFontLabels</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="btnFontLabels">
            <property name="text">
             <string>Font...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="testLabels">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>1 12 35</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>&amp;Results</string>
            </property>
            <property name="buddy">
             <cstring>btnFontResults</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="btnFontResults">
            <property name="text">
             <string>Font...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="testResults">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>日本語能力試験</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Results list</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&amp;Max dictionary results</string>
            </property>
            <property name="buddy">
             <cstring>resultMax</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="resultMax">
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>1000000</number>
            </property>
            <property name="singleStep">
             <number>50</number>
            </property>
            <property name="value">
             <number>10000</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Dictionary paths</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QListWidget" name="dictPaths"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="btnAddPath">
              <property name="text">
               <string>Add...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnDelPath">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OKBtn">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonsCnt</tabstop>
  <tabstop>buttonsCntKana</tabstop>
  <tabstop>btnFontButtons</tabstop>
  <tabstop>btnFontLabels</tabstop>
  <tabstop>btnFontResults</tabstop>
  <tabstop>resultMax</tabstop>
  <tabstop>dictPaths</tabstop>
  <tabstop>btnAddPath</tabstop>
  <tabstop>btnDelPath</tabstop>
  <tabstop>OKBtn</tabstop>
  <tabstop>CancelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>OKBtn</sender>
   <signal>clicked()</signal>
   <receiver>QSettingsDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>571</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>35</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>QSettingsDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
